packages/flutter_driver/lib/src/extension.dart

```diff
@@ -112,16 +110,42 @@ class FlutterDriverExtension {
     }
   }
 
+  Stream<Duration> _onFrameReadyStream;
+  Stream<Duration> get _onFrameReady {
+    if (_onFrameReadyStream == null) {
+      // Lazy-initialize the frame callback because the renderer is not yet
+      // available at the time the extension is registered.
+      StreamController<Duration> frameReadyController = new StreamController<Duration>.broadcast(sync: true);
+      Renderer.instance.addPersistentFrameCallback((Duration timestamp) {
```

Comment:
i mean it's the same object in principle, but the Scheduler is the one that actually implements it